AmazonChimeSDK/AmazonChimeSDK/internal/analytics/DefaultMeetingStatsCollector.swift

```diff
@@ -29,6 +30,8 @@ import Foundation
         return [EventAttributeName.maxVideoTileCount: maxVideoTileCount,
                 EventAttributeName.retryCount: retryCount,
                 EventAttributeName.poorConnectionCount: poorConnectionCount,
+                EventAttributeName.meetingStartDurationMs: meetingStartConnectingTimeMs == 0 ?
```

Comment:
so if meeting failed to start, will this value be negative or should it be 0?

Reply:
It will be reset to 0 in resetMeetingStats()

Reply:
updated